frontend/src/components/SignIn/index.js

```diff
@@ -1,16 +1,50 @@
 import React from 'react';
 
+import * as firebase from 'firebase/app';
+import app from '../Firebase';
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+
+import {VIEW_TRIPS} from '../../constants/routes';
```

Comment:
Same as previous import comment: https://google.github.io/styleguide/jsguide.html#es-module-imports

Reply:
Added the .js suffix to all import statements that imported .js files.